From b10093ff6f9cbf1a5d7d37880dea1c7acd25d6e3 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Fri, 1 Aug 2008 16:34:39 +0800
Subject: [PATCH] ENGR00086181 Disable RNG driver in MX35 default configuration

System halt in loading RNG driver when exeuting reboot.
Disable RNG driver in MX35 default configuration before the root cause is found

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/configs/imx35_3stack_defconfig |    6 ++----
 1 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/imx35_3stack_defconfig b/arch/arm/configs/imx35_3stack_defconfig
index 32c7e8d..0302d51 100644
--- a/arch/arm/configs/imx35_3stack_defconfig
+++ b/arch/arm/configs/imx35_3stack_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.24
-# Thu Jul 31 16:50:58 2008
+# Fri Aug  1 16:25:39 2008
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -1298,9 +1298,7 @@ CONFIG_MXC_MC9SDZ60_POWER=y
 #
 CONFIG_MXC_SECURITY_SCC=y
 # CONFIG_SCC_DEBUG is not set
-CONFIG_MXC_SECURITY_RNG=y
-CONFIG_MXC_RNG_TEST_DRIVER=y
-# CONFIG_MXC_RNG_DEBUG is not set
+# CONFIG_MXC_SECURITY_RNG is not set
 CONFIG_MXC_SECURITY_RTIC=y
 # CONFIG_MXC_RTIC_TEST_DEBUG is not set
 CONFIG_MXC_SECURITY_CORE=y
-- 
1.5.4.4

